contrib/presets/icons/nerd-font.conf

```diff
@@ -0,0 +1,81 @@
+# This preset use symbols from Nerd fonts.
+# https://www.nerdfonts.com/
+# It is required that your terminal emulator use a patched font to see it correctly.
+
+LP_TIME_FORMAT="%Hₕ%Mₘ%Sₛ"
+LP_DELIMITER_KUBECONTEXT_PREFIX=""
+LP_DELIMITER_KUBECONTEXT_SUFFIX=""
```

Comment:
These two do not load for me, where all the others do. Do you have extra extensions besides nerdfont?

Reply:
I thought I only had nerdfont, but it's very possible I have more. I remember having some difficulties finding my default font. I see that the icon (first character of the prefix) is in nerdfont, so I just replaced the other ones.